src/main/java/moneytransferapp/InitiateMoneyTransfer.java

```diff
@@ -0,0 +1,39 @@
+package moneytransferapp;
+
+import io.temporal.client.WorkflowClient;
+import io.temporal.client.WorkflowOptions;
+import io.temporal.serviceclient.WorkflowServiceStubs;
+import java.util.UUID;
+
+// @@@SNIPSTART money-transfer-project-template-java-workflow-initiator
+public class InitiateMoneyTransfer {
+
+    public static void main(String []args) throws Exception {
+        // WorkflowServiceStubs is a gRPC stubs wrapper
+        // That talks to the local Docker instance of the Temporal server.
+        WorkflowServiceStubs service = WorkflowServiceStubs.newInstance();
+        WorkflowOptions options = WorkflowOptions.newBuilder()
+                .setTaskQueue(Shared.MONEY_TRANSFER_TASK_QUEUE)
+                .setWorkflowId("money-transfer-workflow-001")
+                .build();
+        // WorkflowClient can be used to start, signal, query, cancel, and terminate Workflows.
+        WorkflowClient client = WorkflowClient.newInstance(service);
+        // WorkflowFlowStub converts the parameters that are passed to it so that they can be passed to the server.
```

Comment:
I don't understand this comment. Parameters usually refer to method arguments.

Reply:
Went with Ryland's suggestion - LMK what you think.